platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -313,7 +297,20 @@ class HangingProtocolService {
     return protocol;
   }
 
-  _setProtocol(protocol) {
+  public applyProtocol(
+    protocol: HangingProtocol.Protocol,
```

Comment:
It should be possible to apply the protocol by name instead of handing it a full protocol in order to re-apply existing protocols, eg to go back to the previously applied protocol (un-apply the MPR mode).